src/Modules/OrchardCore.Commerce/Assets/Styles/shopping-cart-widget.scss

```diff
@@ -0,0 +1,19 @@
+.widget-shopping-cart-widget {
+    position: absolute;
+    right: 10px;
+    top: 10px;
+}
+
+.shopping-cart-link {
+    position: relative;
+
+    // Making sure widget is always visible.
+    z-index: 9999;
+}
+
+.item-count {
```

Comment:
Why is this class just "item-count"? Seems too generic, where it may suffer from name conflicts. It would be better to reorganize the file to be like 
```scss
.shopping-cart {
    &-widget { ... }
    &-link { ... }
    &-item-count { ... }
}
```